validation/validation_test.go

```diff
@@ -61,40 +74,40 @@ func testValidation(t *testing.T, chain *ChainConfig) {
 // designed to protect downstream software or
 // sanity checking basic consistency conditions.
 func testUniversal(t *testing.T, chain *ChainConfig) {
-	t.Run(GenesisHashTest, func(t *testing.T) { testGenesisHash(t, chain.ChainID) })
-	t.Run(GenesisRPCTest, func(t *testing.T) { testGenesisHashAgainstRPC(t, chain) })
-	t.Run(UniquenessTest, func(t *testing.T) { testIsGloballyUnique(t, chain) })
-	t.Run(ChainIDRPCTest, func(t *testing.T) { testChainIDFromRPC(t, chain) })
-	t.Run(OptimismConfigTest, func(t *testing.T) { testOptimismConfig(t, chain) })
+	t.Run(GenesisHashTest, applyExclusions(chain, testGenesisHash))
+	t.Run(GenesisRPCTest, applyExclusions(chain, testGenesisHashAgainstRPC))
+	t.Run(UniquenessTest, applyExclusions(chain, testIsGloballyUnique))
+	t.Run(ChainIDRPCTest, applyExclusions(chain, testChainIDFromRPC))
+	t.Run(OptimismConfigTest, applyExclusions(chain, testOptimismConfig))
 }
 
 // testStandardCandidate applies to Standard and Standard Candidate Chains.
 func testStandardCandidate(t *testing.T, chain *ChainConfig) {
 	// Standard Config Params
-	t.Run(RollupConfigTest, func(t *testing.T) { testRollupConfig(t, chain) })
-	t.Run(GasTokenTest, func(t *testing.T) { testGasToken(t, chain) })
-	t.Run(ResourceConfigTest, func(t *testing.T) { testResourceConfig(t, chain) })
-	t.Run(GasLimitTest, func(t *testing.T) { testGasLimit(t, chain) })
-	t.Run(GPOParamsTest, func(t *testing.T) { testGasPriceOracleParams(t, chain) })
-	t.Run(StartBlockRPCTest, func(t *testing.T) { testStartBlock(t, chain) })
-	t.Run(SuperchainConfigTest, func(t *testing.T) { testSuperchainConfig(t, chain) })
```

Comment:
this was purposefully moved to `testStandard`?

Reply:
Yes, see PR title.

Reply:
😂